R/numberplate.R

```diff
@@ -0,0 +1,530 @@
+get_numberplate_type <- function(df, registration, firstUsedDate){
+
+
+NumberPlateScheme <- readr::read_csv("NumberPlateScheme.csv")
+for (i in 1:length(df$registration)){
```

Comment:
Some nifty regex could greatly reduce the number of lines in this function. But if it works and, more importantly, has tests, I'm happy accept and look to tidy up after it's in there to keep this moving fast 🚗

Reply:
I’m sure some Regex could - I was trying to keep it human-readable and expandable. Not least to make it more checkable. Mostly cut-paste from the UK numberplate Wikipedia page. I’m going to add it NI etc in due course.